stable/opsportal/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: 1.5.0
 home: https://github.com/mesosphere/charts
 description: OpsPortal Chart
 name: opsportal
-version: 0.3.28
+version: 0.3.29
```

Comment:
you are bumping opsportal too, is that intended ?

Reply:
yes, I added UI bumps to this PR as well